docs-src/docs/guides/walkthrough/deployment-2.md

```diff
@@ -265,12 +265,10 @@ Your data should still be there!
 
 ### Cleanup
 
-If you're finished with the example and don't want to continue with this walkthrough, you can instruct Layer0 to delete the environment and terminate the application.
+When you're finished with the example, instruct Layer0 to delete the environment and terminate the application:
```

Comment:
[nit] "instruct" is a odd word to use here

Reply:
It seems to be used similarly above in some of the argument summary sections. Do those also seem out of place, or is it just this one?